src/Relude/Monad.hs

```diff
@@ -15,9 +15,23 @@ module Relude.Monad
        , module Relude.Monad.Maybe
        , module Relude.Monad.Reexport
        , module Relude.Monad.Trans
+       , chainedTo
        ) where
 
 import Relude.Monad.Either
 import Relude.Monad.Maybe
 import Relude.Monad.Reexport
 import Relude.Monad.Trans
+
+-- $setup
+-- >>> import Relude
+
+{- | For chaining monadic operations in forward applications using '(&)'
+
+>>> Just [ 1 :: Int ] & chainedTo (viaNonEmpty head)
+Just 1
+>>> Nothing & chainedTo (viaNonEmpty head)
+Nothing
+-}
+chainedTo :: Monad m => (a -> m b) -> m a -> m b
+chainedTo = (=<<)
```

Comment:
Basically, the same suggestions as for `appliedTo` function apply to `changedTo` as well 🙂